windows/installer-source/KaliteSetupScript.iss

```diff
@@ -53,15 +50,12 @@ Name: "{group}\{cm:ProgramOnTheWeb,{#MyAppName}}"; Filename: "{#MyAppURL}"
 Name: "{group}\{cm:UninstallProgram,{#MyAppName}}"; Filename: "{uninstallexe}"
 Name: "{commondesktop}\{#MyAppName}"; Filename: "{app}\{#MyAppExeName}"; Tasks: desktopicon ; IconFilename: "{app}\images\logo48.ico"
 
-[Run]
-Filename: "{app}\{#MyAppExeName}"; Description: "{cm:LaunchProgram,{#StringChange(MyAppName, '&', '&&')}}"; Flags: shellexec postinstall skipifsilent
-
 [Dirs]
-Name: "{app}\"; Permissions: everyone-modify
+Name: "{app}\"; Permissions: everyone-readexec

```

Comment:
Keep an eye on the assessmentitems.sqlite database, which I had to do a quickfix for in the .deb -- because I put assessment items in a system wide location, where django then coughed because the sqlite db wasn't readable. This should be fixed in the sdist, anyways.

https://github.com/learningequality/ka-lite/issues/4638